.docker/README.md

```diff
@@ -0,0 +1,78 @@
+# HarbourBridge: Docker Mode
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation and
+migration. This README provides details of HarbourBridge in docker mode. For general HarbourBridge information see
```

Comment:
> provides instructions on how to run HarbourBridge inside a docker.

Reply:
> provides instructions on how to run HarbourBridge inside a docker.

The `Before you begin`, `Pull HarbourBridge's docker image` and `Run the image` sections describe the instructions to run HarbourBridge inside docker.

Reply:
Done.